scripts/monitoring/cron-send-create-app.py

```diff
@@ -86,11 +86,13 @@ def send_metrics(build_ran, create_app, http_code, run_time):
 
     if build_ran == 1:
         ms.add_metric({'openshift.master.app.build.create': create_app})
-        ms.add_metric({'openshift.master.app.build.create.code': http_code})
+        ms.add_metric({'openshift.master.app.create.route_http_failed': route_http_failed})
```

Comment:
these should be `openshift.master.app.build.` as this is in the build section. Though this whole if could be different, but further refactoring is not needed for this PR.

Reply:
I had considered that, but I chose to keep a single Zabbix item for each of the http failure types. There are a couple reasons to keep a single item instead of separating them out into "http failure that happened during build" vs "http failure that happened during a normal app-create":

* We would have 2 counters going for the a single issue, so we wouldn't alert quickly enough. The builds only run every 30 minutes, and each run times out after 9 minutes if a URL is unreachable. So it would take 2 hours to notice route failures and alert us. (Similar to how we alert for build failures today). IMO, the router potentially being offline needs a quicker response than a build failure does.

* An http failure encountered during app-create wouldn't increment the build counter, because it has its own counter.

* We would receive two alerts for the same issue, if the issue continued on for 2 hours. Which would be unnecessary noise.

* Adding an additional two items for each cluster to keep track of means more load on the zabbix server. I would want to justify any additional load being added to the Zabbix server, since it already carries a heavy load.

So by making just one counter (zabbix item) for this, it gets incremented faster and we can tell when something has gone wrong with a router faster.